cli/src/constants/keyboard/keyCodes.ts

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Do we really need to come up with all this ourselves? Is there not some package that defines this stuff for us?